cmd/root.go

```diff
@@ -129,5 +160,6 @@ func init() {
 	RootCmd.PersistentFlags().StringVarP(&to, "to", "t", "home", "Destination location name")
 	RootCmd.Flags().IntVarP(&numResults, "number", "n", 5, "How many commute times do you want?")
 	RootCmd.Flags().IntVarP(&interval, "interval", "i", 15, "How many minutes between each commute prediction?")
+	RootCmd.Flags().StringVarP(&start, "start", "s", "", "Starting commute time (1219:1330 or 1219:0130PM = December 19th at 1:30PM)")
```

Comment:
Should be a persistent flag so we can pass this value into the `commuter view` command. That way when they use `commuter view` it is relevant to the time they specified